e2e/nomostest/portforwarder/port_forwarder.go

```diff
@@ -186,6 +196,12 @@ func (pf *PortForwarder) portForwardToPod(pod string, async bool) error {
 }
 
 func (pf *PortForwarder) portForwardToDeployment(async bool) error {
+	select {
+	case <-pf.ctx.Done():
+		// Don't bother, subprocess already terminated/terminating
+		return errors.Wrap(pf.ctx.Err(), "failed to start port forward process")
```

Comment:
error message seems potentially misleading, maybe something like `context already closed`?

Reply:
the context error it wraps will say "cancelled" or "deadline exceeded"